packages/desktop/src/renderer/components/widgets/channels/ChannelInput/ChannelInput.tsx

```diff
@@ -470,8 +476,20 @@ export const ChannelInputComponent: React.FC<ChannelInputProps> = ({
                   html={sanitizedHtml}
                   onChange={onChangeCb}
                   onKeyDown={onKeyDownCb}
-                  onPaste={(e) => {
+                  onPaste={async (e) => {
                     e.preventDefault()
+
+                    const files = e.clipboardData.files
+                    for (let i = 0; i < files.length; i++) {
+                      const fileNameSplited = files[i].name.split('.')
+                      const fileExtension = fileNameSplited[fileNameSplited.length - 1]
+                      const extensions = ['png', 'jpg', 'jpeg']
+                      if (extensions.includes(fileExtension)) {
+                        const arrayBuffer = await files[i].arrayBuffer()
+                        imagesFromClipboard(arrayBuffer, fileExtension)
```

Comment:
It's a function, name suggests that's some data structure containing images, maybe better name it handleClipboard